packages/wonder-blocks-form/src/components/labeled-text-field.tsx

```diff
@@ -296,4 +294,4 @@ type ExportProps = Omit<
 
 export default React.forwardRef<HTMLInputElement, ExportProps>((props, ref) => (
     <LabeledTextField {...props} forwardedRef={ref} />
-)) as Flow.AbstractComponent<ExportProps, HTMLInputElement>;
+)) as React.ForwardRefExoticComponent<ExportProps & React.RefAttributes<HTMLInputElement>>;
```

Comment:
This is based on the return type of `React.forwardRef`.  We might be able to drop it since these type params are already being passed to `React.forwardRef` itself.